R/setkey.R

```diff
@@ -51,14 +51,9 @@ setkeyv = function(x, cols, verbose=getOption("datatable.verbose"), physical=TRU
   }
   if (identical(cols,"")) stop("cols is the empty string. Use NULL to remove the key.")
   if (!all(nzchar(cols))) stop("cols contains some blanks.")
-  if (!length(cols)) {
```

Comment:
this `if` is redundant & contradictory -- just a few lines earlier (L47) has `if (!length(cols))` as a `warning`. I guess this branch was some leftover or copy/paste from `setkey`... revealed by Codecov